docs/core/testing/index.md

```diff
@@ -35,47 +35,64 @@ Keep in mind there are [best practices](unit-testing-best-practices.md) for writ
 
 ## Testing tools
 
-.NET is a multi-language development platform, and you can write various test types for [C#](../../csharp/index.yml), [F#](../../fsharp/index.yml), and [Visual Basic](../../visual-basic/index.yml). For each of these languages, you can choose between several test frameworks.
+When running tests in .NET, there are two components involved. The test platform and the test framework.
 
-### xUnit
+### Test platforms
 
-[xUnit](https://xunit.net) is a free, open-source, community-focused unit testing tool for .NET. The original inventor of NUnit v2 wrote xUnit.net. xUnit.net is the latest technology for unit testing .NET apps. It also works with ReSharper, CodeRush, and TestDriven.NET. xUnit.net is a project of the [.NET Foundation](https://dotnetfoundation.org) and operates under its code of conduct.
+The test platform is the engine that runs the tests and acts as a communication channel with IDEs. For example, the IDE (e.g, Test Explorer) can send a discovery request to the test platform so that the IDE is able to show the tests you have. The test platform responds, in some way, back to the IDE with the tests it found. Similar communication happens for test execution.
```

Comment:
```suggestion
The test platform is the engine that runs the tests and acts as a communication channel with IDEs. For example, an IDE such as Test Explorer can send a discovery request to the test platform so that it can display the available tests. The test platform responds back to the IDE with the tests it found. Similar communication happens for test execution.
```

Reply:
I'm realizing now the wording isn't super accurate. Test Explorer is a component of Visual Studio IDE that's involved in the communication. Giving "Test Explorer" as example of "IDE" is misleading as TE itself isn't an IDE. Not sure how to best reword that.

Reply:
```suggestion
The test platform is the engine that runs the tests and acts as a communication channel with IDEs. For example, Visual Studio can send a discovery request to the test platform so that it can display the available tests in Test Explorer. The test platform responds back to the IDE with the tests it found. Similar communication happens for test execution.
```